ui/balance_druid/presets.ts

```diff
@@ -278,7 +278,7 @@ name: 'Basic P3 APL',
 rotation: SavedRotation.create({
 	specRotationOptionsJson: BalanceDruidRotation.toJsonString(DefaultRotation),
 	rotation: APLRotation.fromJsonString(`{
-      "type": "TypeAPL",
+      "type": 3,
```

Comment:
turns out the string there was not actually parsing correctly, causing the 'IsUsingAPL' flag to not be set correctly.